include/onnxruntime/core/common/common.h

```diff
@@ -74,6 +74,9 @@ using common::Status;
   static_cast<void>(fn)
 
 std::vector<std::string> GetStackTrace();
+// these is a helper function that gets defined by platform/Telemetry
+void LogRuntimeError(uint32_t sessionId, const common::Status& status, const char* file,
```

Comment:
session_id

Reply:
Should this method be const too?

Reply:
it's a not a member function, this one is a global helper.   we only get to use "const" on member functions to denote they can't change member vars. right?

---
In reply to: [335302396](https://github.com/microsoft/onnxruntime/pull/2109#discussion_r335302396) [](ancestors = 335302396)

Reply:
ah, didn't realize that it wasn't a member function.